module/modmanager/manager.go

```diff
@@ -714,7 +714,7 @@ func (m *module) registerResources(mgr modmaninterface.ModuleManager, logger gol
 			}
 		case api.API.IsService():
 			for _, model := range models {
-				logger.Debugw("registering service from module", "module", m.name, "API", api.API, "model", model)
+				logger.Infow("registering service from module", "module", m.name, "API", api.API, "model", model)
```

Comment:
Changed this as part of another small ticket in rdk: https://viam.atlassian.net/browse/APP-2700?atlOrigin=eyJpIjoiYTNlZDUyMzllYzc3NDU0YjgxYTUxNjdiNjc5Y2I5YTYiLCJwIjoiaiJ9